src/Symfony/Component/HttpKernel/Profiler/Profiler.php

```diff
@@ -152,6 +155,10 @@ public function collect(Request $request, Response $response, \Throwable $except
             $profile->setIp('Unknown');
         }
 
+        if ($request->attributes->has('_virtual')) {
+            $profile->setVirtualType($request->attributes->get('_virtual'));
```

Comment:
let's use `_virtual_type`?
```suggestion
            $profile->setVirtualType($request->attributes->get('_virtual_type'));
```

Reply:
Done.